redash/handlers/query_results.py

```diff
@@ -37,6 +38,7 @@
     serialize_job,
 )
 
+WRITER_ENCODING = os.environ.get("REDASH_CSV_WRITER_ENCODING", "utf-8")
```

Comment:
We should put it in the settings.

Reply:
@junnplus I guess you mean the list of environment variable strings?

https://redash.io/help/open-source/admin-guide/env-vars-settings

If so, then definitely yes. :smile:

Reply:
not website, jun mean have to change to file https://github.com/getredash/redash/blob/7f4ade5f1ff95f4357f60acb07b0655a33527972/redash/settings/__init__.py